sklearn/linear_model/tests/test_quantile.py

```diff
@@ -0,0 +1,102 @@
+# Authors: David Dale dale.david@mail.ru
+# License: TBD
```

Comment:
We have the same license across the code base...

Reply:
Fixed it